NetscapeBookmarkParser.php

```diff
@@ -37,6 +51,15 @@ public function __construct(
             $this->defaultTags = array();
         }
         $this->defaultPub = $defaultPub;
+
+        $logsDir = ! empty($logsDir) ? $logsDir : __DIR__ . '/logs/';
+        $this->logger = new Logger(
+            $logsDir,
+            $logLevel,
+            array(
+                'filename' => 'import.log',
```

Comment:
You're right. Actually I don't know why I used this way, since the logger is PSR compliant, I'll just add a setter.